### DISCUSSION TITLE
How can activate continious jog ?

### DISCUSSION THREAD

**Discussion by rmaltais** [terjeio/ioSender#308] [General]
Hi,

You talk about continious jog but i can't see how do that ? My jog only go with a specific lenght.

thank

**Comment by rmaltais** [comment]
Ok sorry i found it in reply somewhere. Argument on startup command. 

-jogmod <mode>. 

See in wiki

**Reply by terjeio** [reply]
This [wiki page](https://github.com/terjeio/ioSender/wiki/Setup-and-configuration) has the info.
